daemon/stats/collector.go

```diff
@@ -89,7 +90,10 @@ func (s *Collector) Run() {
 		for _, pair := range pairs {
 			stats, err := s.supervisor.GetContainerStats(pair.container)
 			if err != nil {
-				if _, ok := err.(notRunningErr); !ok {
+				// if we get "container not found" error from containerd, it's possibly
+				// because that this container has already been stopped. it will be ok to
+				// ignore this error and just return an empty stats.
+				if _, ok := err.(notRunningErr); !ok && !strings.Contains(err.Error(), "container not found") {
```

Comment:
Could we make the `container not found` error a typed error (like we do for `notRunningErr`)?